.github/scripts/run_consensus.sh

```diff
@@ -2,21 +2,51 @@
 
 set -e
 
+# change it when increasing number of node containers
+export NODE_COUNT=5
+
 mkdir -p docker/data/
 
-# source account ids
-source docker/env
+function generate_authorities {
+  local authorities_count="$1"
+
+  echo "Generating ${authorities_count} authorities accounts ids..." >&2
+  declare -a account_ids
+  for node_index in $(seq 0 $((authorities_count - 1))); do
+    echo "Generating authority ${node_index} from key //${node_index}" >&2
+    account_ids+=($(docker run --rm -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" aleph-node:latest -c "aleph-node key inspect //$node_index | grep \"SS58 Address:\" | awk \"{print \\\$3;}\""))
```

Comment:
could we specify the image (export NODE_IMAGE=aleph-node:latest) as a top level var as well for all commands that call to the dockerized node?

This would make it easier to change it in one place if testing against some other version (say testnet or smartnet version)

Reply:
another solution `docker tag <image_id> aleph-node:latest`

Reply:
I would forget the next day I have a image tagged differently and spend hours debugging :rofl:

Reply:
Will do